h2o-py/h2o/model/coxph.py

```diff
@@ -12,3 +12,13 @@ def formula(self):
     def coefficients_table(self):
         """Coefficients table."""
         return self._model_json["output"]["coefficients_table"]
+
+    def summary(self):
```

Comment:
Is it ok, it overloads the `summary` method from the `ModelBase`? 
If there is nothing in `output["model_summary"]`, it is ok. :+1: